src/pokemon_storage_system.c

```diff
@@ -231,7 +231,7 @@ struct PokemonStorageSystemData
     struct Sprite *field_D94;
     struct Sprite *field_D98[2];
     u16 *field_DA0;
-    struct PokemonMarkMenu field_DA4;
+    struct PokemonMarkMenu monMarks;
```

Comment:
I think `markMenu` would be better than `monMarks`.

Reply:
I based the name on from what appeared in `pokenav_conditions_2.c`. Should that also be changed?